Lightprotocol/light-protocol
# test: account compression

changes:
- add associated queue and tree check to update address Merkle tree
- init address merkle tree: add standard config checks at init
- test: failing tests update address Merkle tree
- test: append
- assert_rpc_error returns Result<(), RpcError>
- add NewElementNextIndexMismatch error
- add checks for default value initialization in Address Merkle tree
- add check in insert into queue to throw an error on empty input values
- merkle_tree_test
  - moved all helper functions to the bottom -> diff is awful to look at
- add failing test for NewElementNextIndexMismatch in indexedMerkle tree
- fix address Merkle tree meta data assert (make index variable)
